---
title: "20160117 GSE32719 FL_HHMI15 Aging in HSC" 
author: "h qin"
date: "January 17, 2016"
output: html_document
---

GSE32719
http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32719

```{r, message=FALSE}
rm(list=ls())
setwd("~/github/FL_HHMI15/project1.GSE32719")
library(Biobase)
library(GEOquery)
library(limma)
library(foreach)
library(doMC)
```

load platform data from bioconductor
http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32719
GPL570	[HG-U133_Plus_2] Affymetrix Human Genome U133 Plus 2.0 Array
https://bioconductor.org/packages/release/data/annotation/html/hgu133plus2.db.html


Load series and platform data from GEO
# gset <- getGEO("GSE32719", GSEMatrix =TRUE)

```{r}
 gset <- getGEO(filename="GSE32719_series_matrix.txt.gz")
# make proper column names to match toptable 
# fvarLabels(gset) <- make.names(fvarLabels(gset))
```


Check exression levels. I need to normalize them. !!!!
```{r}
ex= exprs(gset)
apply( ex, 2, median)
means = apply( ex, 2, mean)
means
boxplot( ex )
```

Normalize them by total intensity
```{r}
scale = max(means)
for(i in 1:length(ex[1,])){
  ex[,i] = ex[,i] * scale/ means[i]
}
apply( ex, 2, mean) / scale
apply( ex, 2, median) /scale
boxplot( ex )
```

Study the structure of gset
```{r, comment=NA}
str(gset)
#gset@phenoData@data
experimental_design = gset@phenoData@data
experimental_design[1:3,]
# gset@phenoData@varMetadata
# experimental_design[, "source_name_ch1"][1:10]
experimental_design[1:3,  c("title", "source_name_ch1")]
unique( experimental_design$source_name_ch1 )
```



```{r}
sample.letters = as.factor( LETTERS[1:length(sample.names)]) 
names(sample.letters) = sample.names
```


Using 'limma' for differential expression
```{r}
# set up the data and proceed with analysis
sample.letters[ experimental_design$source_name_ch1 ]
fl <- as.factor( sample.letters[experimental_design$source_name_ch1 ] )
gset$description <- fl
design <- model.matrix(~ description + 0, gset)
colnames(design) <- levels(fl)
fit <- lmFit(gset, design)
#A-B-C does not look right? 
cont.matrix <- makeContrasts(A-B-C, levels=design)
cont.matrix
fit2 <- contrasts.fit(fit, cont.matrix)
fit2 <- eBayes(fit2, 0.01)
tT <- topTable(fit2, adjust="fdr", sort.by="B", number=length(fit2))
```


Load NCBI platform annotation
```{r, message=FALSE}
gpl <- annotation(gset)
platf <- getGEO(gpl, AnnotGPL=TRUE)
ncbifd <- data.frame(attr(dataTable(platf), "table"))
```


```{r}
# replace original platform annotation
tT <- tT[setdiff(colnames(tT), setdiff(fvarLabels(gset), "ID"))]
tT <- merge(tT, ncbifd, by="ID")
tT <- tT[order(tT$P.Value), ]  # restore correct order

tT <- subset(tT, select=c("ID","adj.P.Val","P.Value","t","B","logFC","Gene.symbol","Gene.title"))
tT[1:10, ]
```

```{r, message=FALSE}
write.table(tT, file="__tmp.tsv", row.names=F, sep="\t")
```

Regression for age-dependent changes 
```{r}
sample.names = unique(experimental_design$source_name_ch1)
mylevels = c(1,2,3)
names(mylevels) = sample.names
gset.levels = mylevels[ experimental_design$source_name_ch1 ]
my.pvalues = numeric(length=length(ex[,1]))
# x2 = foreach(i=1:(3*2), .combine='cbind') %dopar% sqrt(i)
registerDoMC(cores=4)
#my.pvalues = foreach( i = 1:length(ex[,1]), .combine='rbind') %dopar% {
my.pvalues = foreach( i = 1:100, .combine='rbind') %dopar% {
  m = lm( ex[i,] ~ gset.levels )
  sm = summary(m)
  pf(sm$fstatistic[1], sm$fstatistic[2], sm$fstatistic[3], lower.tail = FALSE)
}
names(my.pvalues) = row.names(gset@assayData$exprs)
```

Find signicant p-values
```{r}
hist(my.pvalues)
summary(my.pvalues)
my.pvalues.BH = p.adjust(my.pvalues, "BH")
hist(my.pvalues.BH)
summary(my.pvalues.BH)
sig= my.pvalues.BH[my.pvalues.BH < 0.05]
names(sig)
ncbifd[ ncbifd$ID == names(sig),  ]
```



Load bioconductor annotation
```{r, message=FALSE }
# source("https://bioconductor.org/biocLite.R")
# biocLite("hgu133plus2.db")
# library("hgu133plus2.db")
# help(package="hgu133plus2.db")
# ls("package:hgu133plus2.db")
# x <- hgu133plus2GENENAME
# x <- hgu133plus2SYMBOL
# mapped_probes <- mappedkeys(x)
# xx <- as.list(x[mapped_probes])
# xx[1:5]
```
